checkpoint/checkpoint_compare.sh

```diff
@@ -0,0 +1,27 @@
+C1=$1
+C2=$2
+MENTOR=clint
+TEST_SET=testing_data_full.csv
+
+CHECKPOINT_ROOT=$(pwd)
+PROJECT_ROOT=$(dirname ${CHECKPOINT_ROOT})
+
+DOCKER_ACCOUNT=uscictdocker
+DOCKER_REPO=mentorpal-classifier
+DOCKER_TAG=latest
+DOCKER_IMAGE=${DOCKER_ACCOUNT}/${DOCKER_REPO}:${DOCKER_TAG}
+
+docker build -t ${DOCKER_IMAGE} .
```

Comment:
In the ideal case what we want to do here is *not* build a new docker image, but rather use the same mentorpal-classifier docker image that the production app uses